schemas/stsci.edu/asdf/unit/quantity-1.1.0.yaml

```diff
@@ -21,6 +18,7 @@ properties:
         items:
           type: number
```

Comment:
Yes, you are right. The list will be converted to ndarray by `Quantity`. Then is the `array` option necessary at all?